lib/classes/Utils.js

```diff
@@ -30,35 +32,19 @@ class Utils {
   }
 
   fileExistsSync(filePath) {
-    try {
-      const stats = fse.lstatSync(filePath);
-      return stats.isFile();
-    } catch (e) {
-      return false;
-    }
+    // console.log('this.serverless.utils.fileExistsSync being depricated in a future release')
+    // console.log('Please explicitly declare your plugin dependancies')
```

Comment:
This will be deprecated since we're moving towards a more functional codebase in the future where we can easily test all the different pieces independently w/o having to pass around this huge `serverless` object.

I believe those are preparations so that this can be commented out once `v2` will be around the corner.